### PR TITLE
various updates for 0.27

### PR DESCRIPTION
**Pull request by timmysilv**
docs updates while validating changes for the release

### CODE REVIEW

**Comment by rmoyard** [comment]
Those changes https://github.com/PennyLaneAI/pennylane/pull/3321 to be added here.

File: pennylane/gradients/jvp.py [hunk 1]
```diff
@@ -274,7 +309,56 @@ def batch_jvp(tapes, tangents, gradient_fn, shots=None, reduction="append", grad
...
+
+    Both tapes share the same circuit ansatz, but have different measurement outputs.
+
+    We can use the ``batch_jvp`` function to compute the Jacobian vector product,
+    given a list of tangents ``tangent``:
+
+    >>> tangent_0 = [jax.numpy.array(1.0), jax.numpy.array(1.0), jax.numpy.array(1.0), jax.numpy.array(1.0), jax.numpy.array(1.0), jax.numpy.array(1.0)]
+    >>> tangent_1 = [jax.numpy.array(1.0), jax.numpy.array(1.0), jax.numpy.array(1.0), jax.numpy.array(1.0), jax.numpy.array(1.0), jax.numpy.array(1.0)]
+    >>> tangents = [tangent_0, tangent_1]
+    >>> vjp_tapes, fn = qml.gradients.batch_vjp(tapes, dys, qml.gradients.param_shift)
```
Comment (lillian542) [inline pennylane/gradients/jvp.py]: What is `dys` here?

Comment (rmoyard) [reply pennylane/gradients/jvp.py]: I think I removed this line on the latest commit :)

File: pennylane/gradients/jvp.py [hunk 2]
```diff
@@ -270,7 +309,55 @@ def batch_jvp(tapes, tangents, gradient_fn, shots=None, reduction="append", grad
...
+    Executing the JVP tapes, and applying the processing function:
+
+    >>> jvp_tapes, fn = qml.gradients.batch_jvp(tapes, tangents, qml.gradients.param_shift)
+
+    >>> dev = qml.device("default.qubit", wires=2)
+    >>> jvps = fn(dev.batch_execute(jvp_tapes))
+    >>> jvps
+    [(DeviceArray(-0.62073976, dtype=float32), DeviceArray([-0.3259707 ,  0.32597077], dtype=float32)), DeviceArray(-0.6900841, dtype=float32)]
+
+    We have two JVPs; one per tape. Each one corresponds to the shape of the output of respective tape
```
Comment (rmoyard) [inline pennylane/gradients/jvp.py:360]:
```suggestion
    We have two JVPs; one per tape. Each one corresponds to the shape of the output of their respective tape.
```

**Review by rmoyard — APPROVED** [review]
💯
